backend/app/api/endpoints/adapter/subscription_follows.py

```
@@ -41,19 +43,23 @@
 @router.post("/{subscription_id}/follow", status_code=status.HTTP_200_OK)
 def follow_subscription(
     subscription_id: int,
+    request: FollowSubscriptionRequest = Body(
+        default_factory=FollowSubscriptionRequest
+    ),
```

Ruff (0.14.14)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)